### PR TITLE
Integration test persistent storage (berkeley)

### PR DESCRIPTION
**Pull request by dkijania**
Explain your changes:

Added option to enable persistence of entire working directory of each pod in terraform definition. Currently test executive always use persistence

Explain how you tested your changes:

I Run test executive test (payments) and check if storages are created. I created example file in pods and restarted mina processes. Files stay like before restart

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: src/lib/integration_test_cloud_engine/kubernetes_network.ml [hunk 1]
```diff
@@ -78,6 +78,19 @@ module Node = struct
       @ [ "exec"; "-c"; container_id; "-i"; pod_id; "--" ]
       @ cmd )
 
+  let run_in_pod ?(exit_code = 10) ?override_with_pod_id ~cmd t =
```
Comment (QuiteStochastic) [inline src/lib/integration_test_cloud_engine/kubernetes_network.ml:81]: what is the difference between this function and `run_in_container`?  from what I can tell, `run_in_container` uses `kubectl exec -c <container_id> -i <pod_id>`, while this new `run_in_pod` function simply omits the `-c` flag.  doesn't that just mean it gets run in the default container?  I think generally it is good to be explicit as to which container you're running things in

File: src/lib/logger/logger.mli [hunk 2]
```diff
@@ -162,6 +163,8 @@ val spam :
   -> ('a, unit, string, unit) format4
   -> 'a
 
+val span : _ log_function
```
Comment (QuiteStochastic) [inline src/lib/logger/logger.mli:166]: what is the `span` level exactly?

File: src/lib/integration_test_cloud_engine/kubernetes_network.ml (hunk 1 above)
Comment (dkijania) [reply src/lib/integration_test_cloud_engine/kubernetes_network.ml]: Apologies this commit was some leftover and was reverted

File: src/lib/logger/logger.mli (hunk 2 above)
Comment (dkijania) [reply src/lib/logger/logger.mli]: Apologies this commit was some leftover and was reverted

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by nholland94 — APPROVED** [review]

**Comment by nholland94** [comment]
!approved-for-mainnet

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**